distributionviewer/core/static/js/app/components/containers/chart-list-container.js

```diff
@@ -12,7 +12,7 @@ export class ChartListContainer extends React.Component {
     this.state = {numCharts: 0};
   }
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
Nah I am probably jumping the gun on it. I'd rather try it stricter and then see if it's viable going forward.